eslint/index.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
     "no-unused-vars": ["error", { ignoreRestSiblings: true }],
     "react/jsx-no-target-blank": "warn",
     "react-hooks/rules-of-hooks": "error",
+    "react-hooks/exhaustive-deps": "warn",
```

Comment:
Are you sure we need this rule?

Last time I've added it, noone was happy with workflow of this rule and it was removed.

Reply:
@avg206 removed exhaustive hooks as they add stuff while --fix